src/state/persisted/legacy.ts

```diff
@@ -61,7 +61,7 @@ type LegacySchema = {
     copiedInvites: string[]
   }
   mutedThreads: {uris: string[]}
-  reminders: {lastEmailConfirm: string}
+  reminders: {lastEmailConfirm?: string}
```

Comment:
Small type update. Access below was using optional chaining, but this is more correct.